yutuf_serch.py

```diff
@@ -0,0 +1,27 @@
+import requests
+from googleapiclient.discovery import build
+
+def get_youtube_video_ids(api_key, search_query):
+    youtube = build('youtube', 'v3', developerKey=api_key)
+
+    search_response = youtube.search().list(
+        q=search_query,
+        part='id',
+        type='video',
+        maxResults=15
+    ).execute()
+
+    video_ids = [item['id']['videoId'] for item in search_response['items']]
+    return video_ids
+
+if __name__ == "__main__":
+    api_key = "AIzaSyASitxw_ki1EAc0jmgipZ77p8Nb0ig4ICc"
+    search_query = input("Please enter the query, pănă nutz dau două: ")
```

Comment:
The input prompt must be friendly and in one international language standard

Reply:
e promptu meu, il fac CUM vreau